gammapy/catalog/tests/test_fermi.py

```diff
@@ -439,3 +440,9 @@ def test_select_source_classes(self):
 
         selection = self.cat.select_source_class("PSR")
         assert len(selection.table) == 53
+
+def test_4fgl():
+    cat = SourceCatalog4FGL()
+    source = cat[188]
+    spectral_model = source.spectral_model
+    print(spectral_model)
```

Comment:
Suggest to parametrise this test (pick 3 sources for 3 spectral models you support), as in 3FGL test_spectral_model test above. Try to do the same as `TestFermi3FGLObject`.

Reply:
done!